### ISSUE TITLE
[FEATURE] Need a mechanism for users to add alternative text for images uploaded. 

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#1964]
**Is your feature request related to a problem? Please describe.**
(Accessibility issue with image uploading, no images have alternative text)
All non-text content that is presented to the user has a text alternative that serves the equivalent purpose.
[WCAG 1.1.1 Non-Text Content (A)](https://www.w3.org/TR/WCAG21/#dfn-non-text-content)

**Describe the solution you'd like**
Currently, the uploader being used does not have a required field for "Alternative text" there for images are not being pronounced properly to users of assistive technology. It is inaccessible. Please add a field that is required, for "Alternative text" that explains to users (possibly with help text) that this is to describe the image for those that cannot see it. 

**Describe alternatives you've considered**
Add a field or create a custom uploader. 

**Additional context**
This is anywhere in the app where user may upload a photo, or even in the events manager in the file upload field, there needs to be a field for alt text too.

**Comment by acovher2** [comment]
@pmarkhennessy @TodorBachvarov any update on this feature request? 
@knguyenjahiel

**Comment by acovher2** [comment]
@pmarkhennessy  Due to the ADA title II changes and deadline of April 2026, I wanted to ensure this gets on the list and radar as we need to be compliant. This was on the PPT I sent you.

**Comment by acovher2** [comment]
Closing as #4836 Handles this.
